brownie/network/account.py

```diff
@@ -59,6 +59,19 @@ def _reset(self) -> None:
             self._accounts = [Account(i) for i in web3.eth.accounts]
         except Exception:
             pass
+
+        # Check if accounts were manually unlocked and add them
+        try:
+            unlocked_accounts = CONFIG.active_network["cmd_settings"]["unlock"]
+            if not isinstance(unlocked_accounts, list):
+                unlocked_accounts = [unlocked_accounts]
+            for address in unlocked_accounts:
+                if isinstance(address, int):
+                    address = HexBytes(address.to_bytes(20, "big")).hex()
+                self._accounts.append(Account(address))
```

Comment:
What if the user unlocks an account that's already unlocked via their mnemonic, or includes the same account more than once in the list?